server/camcops_server/cc_modules/cc_forms.py

```diff
@@ -1016,22 +1017,24 @@ def after_bind(self, node: SchemaNode, kw: Dict[str, Any]) -> None:
         self.title = _("ID# value")
 
 
-class OptionalIdNumValue(MandatoryIdNumValue):
-    """
-    Optional node to capture an ID number value.
-    """
-    default = None
-    missing = None
-
-    @staticmethod
-    def schema_type() -> SchemaType:
-        return AllowNoneType(Integer())
+# class OptionalIdNumValue(MandatoryIdNumValue):
+#     """
```

Comment:
Do we need this class? Can we remove it altogether?

Reply:
Can remove!